### ISSUE TITLE
bug(mat-select): Standard appearance select's ripple is not visible in high contrast mode

### ISSUE THREAD

**Issue by dwaynelavon** [angular/components#21780]
#### Reproduction

Stackblitz: https://stackblitz.com/edit/angular-lh4cau?file=src/app/select-overview-example.html

Steps to reproduce:
1. Turn on high contrast mode
2.  Open Stackblitz and tab to the mat-select 
 
 
#### Expected Behavior

What behavior were you expecting to see?
Focus ripple should be visible.

#### Actual Behavior

What behavior did you actually see?
Focus ripple is not visible. Only seems to be an issue with `appearance="standard"`. Legacy works fine

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows with High Contrast Mode
